applications/DemStructuresCouplingApplication/custom_python/add_custom_utilities_to_python.cpp

```diff
@@ -60,7 +60,7 @@ namespace Kratos {
                 .def("ExecuteFinalizeSolutionStep", &StressFailureCheckUtilities::ExecuteFinalizeSolutionStep)
             ;
 
-            class_<PostProcessUtilities, PostProcessUtilities::Pointer>(m, "PostProcessUtilities")
+            class_<PostProcessUtilities,PostProcessUtilities::Pointer>(m, "PostProcessUtilities", module_local())
```

Comment:
what does this do?

Reply:
Hi @philbucher. This is a temporal solution that @maceligueta and me have found for a problem regarding the PostProcessUtilities. Without adding this, the code breaks... Maybe @maceligueta can add some info

Reply:
Hi @philbucher , this is an important matter that I was planning to post on a new issue. Basically, when you have utilities with the same name in different applications you may get the following error:
from KratosDemStructuresCouplingApplication import *
ImportError: generic_type: type "PostProcessUtilities" is already registered!``
This is kind of mind-blowing, because the examples we tried were not doing any 'import *' and the utilities were clearly encapsulated in each respective application. Also @JoseMGonzalez-UPC  experienced this in his private branch for drilling-beam-application. We requested @roigcarlo 's help and finally this solution worked. 
This made me think that we will come across with this issue frequently in the future, when people develop more and more couplings between applications.
My proposal is to add 'module_local()' in all applications by default. 
Maybe @roigcarlo can bring his opinion to clarify.

Reply:
Hm this is interesting

so lets say util `Abc` exists in `XApplication` and `YApplication`, then the following would still fail?
~~~py
from XApplication import Abc as x_abc
from YApplication import Abc as x_abc
~~~

and if you add `module_local` then it works?

Reply:
Well, I guess this example is weirder because you are overwriting x_abc.
The problem we detected was raised at
```
import XApplication as X
import YApplication as Y  #<--- ERROR
```
where both applications have an utility called `Abc`

Reply:
> Well, I guess this example is weirder because you are overwriting x_abc.

ok I did not intend that, was typo

But thanks for clarifying

adding it to the todo of the committee

Reply:
I just wanted to make clear that, even though in the example I posted I wrote
```
from KratosDemStructuresCouplingApplication import *
ImportError: generic_type: type "PostProcessUtilities" is already registered!
```
with @roigcarlo  and @JoseMGonzalez-UPC  we tried the standard import
`import KratosDemStructuresCouplingApplication as DemStruct`
and the problem was still there.